### ISSUE TITLE
Test visual design of meta menu

### ISSUE THREAD

**Issue by elbotho** [serlo/frontend#169]
from @simonkoehl (moved this here so we can close the old issue)

> […] How about getting rid of the frame:
![Neue Sub-Navi](https://user-images.githubusercontent.com/3379377/81216810-d1e0c580-8fdb-11ea-8658-5e19114097b9.png)

> @elbotho are you sure it is more obvious with the frame? My first intuition with the frame was "button". The fade out effect at the right hand side is the only thing that indicates the slider. And this effect is weaker with the frame imo. But I can do some testing with 3-5 people here in the house to disprove that, if you provide me with a link?

**Comment by Entkenntnis** [comment]
@elbotho @simonkoehl Any progress on this one? I have no strong opinion in either direction. I am fine with the border.

More annoying is the big scrollbar on desktop, but this another issue.

**Comment by Entkenntnis** [comment]
Latest update: We would need some testing with users here, before we can come to an decision.

**Comment by elbotho** [comment]
Okay, I redesigned it to make it more clear that is interactive (also removed the border).
Since I prefer the new version, I will close this.
Please open new issue for further feedback.

(ping @simonkoehl)
